package.json

```diff
@@ -3,22 +3,22 @@
   "bugs": "https://github.com/jaredpalmer/razzle/issues",
   "homepage": "https://github.com/jaredpalmer/razzle#readme",
   "devDependencies": {
-    "eslint": "^4.6.1",
-    "husky": "^0.14.3",
-    "jest": "^24.9.0",
-    "lerna": "2.10.1",
-    "lerna-changelog": "^0.7.0",
-    "lint-staged": "^4.0.0",
-    "prettier": "^1.4.4",
-    "ps-tree": "1.1.0",
-    "semver": "5.5.1",
-    "shelljs": "^0.8.2"
+    "eslint": "^6.8.0",
+    "husky": "^4.2.3",
```

Comment:
Make sure to update breaking `husky` changes. You can see the example for 2.7 here: https://github.com/jaredpalmer/razzle/pull/1059

Reply:
❤️